agents/pfeiffer_tpg366/pfeiffer_tpg366_agent.py

```diff
@@ -0,0 +1,50 @@
+#script to log and readout pfeiffer TPG 366 gauge contoller
+#via Ethernet connection
+#Zhilei X., Tanay B.
+
+import socket
+import numpy as np
+
+BUFF_SIZE = 128
+ENQ = '\x05'
+IP_ADDRESS = '10.10.10.20'
+
+class pfeiffer:
+    def __init__(self, ip_address=IP_ADDRESS, port=8000, timeout=10):
+        self.ip_address = ip_address
+        self.port = port
+        
+        self.comm = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        self.comm.connect((self.ip_address, self.port))
+        self.comm.settimeout(timeout)
+    
+    def read_pressure(self, ch_no):
```

Comment:
Docstring formatting should be not as single comments (#), but as a docstring surrounded by sets of triple-quotes ("""). The SO dev guide recommends the Numpy style, example here: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html#example-numpy

Please reformat.

Reply:
Fixed

Reply:
Formatting again. I would reformat to:
```
"""Function to measure the pressure of one given channel.

Args:
    ch_no (int): The channel to be measured (e.g. 1-6)

Returns:
    float: The measured pressure

"""
```